File: find_multiples.py
def find_multiples(nums, count):
  return_arr = []
  for num in nums:
    if num % count == 0:
      # print("num: ", num)
      return_arr.append(num)
      # print("return_arr", return_arr)
  return return_arr


print(find_multiples([1, 2, 3], 2)) # 2

File: findMultiples.js
function findMultiples(arr, count) {
  let freqObj = freqCounter(arr);
  let retArr = [];
  for (let key in freqObj) {
    if (freqObj[key] >= count) {
      retArr.push(+key);
    }
  }
  return retArr;
}



function freqCounter(arr) {
  let obj = {};
  for (let i = 0; i < arr.length; i++) {
    if (obj[arr[i]] === undefined) {
      obj[arr[i]] = 1;
    } else {
      obj[arr[i]] += 1;
    }
  } 
  return obj;
}
